cloudfour/fluidlint
# Initial

# Testing

- Install
  ```
  npm ci
- Set it up so that your local `fluidlint` version is available globally:
  ```
  npm link
  ```
- Create a liquid file that is invalid. Confirm that running `fluidlint` prints the errors
